### ISSUE TITLE
Tool naming

### ISSUE THREAD

**Issue by vreuter** [databio/dnameth_pipelines#22]
Why does a script for which the name suggests RNA run here? What's this tool do?
```
`python /sfs/qumulo/qhome/vpr9v/code/dnameth_pipelines/src/tools/bisulfiteReadFiltering_forRNA.py
```

**Comment by nsheff** [comment]
@jklughammer any recollection here?

**Comment by jklughammer** [comment]
yeah, we created this for the core-seq protocol, which was supposed to assesses DNA methylation and RNA   in the same sequencing library. In RNA mode this script was supposed to identify the reads that originated from RNA. I guess it is not really needed in a standard DNA methylation pipeline.

**Comment by vreuter** [comment]
OK cool, that makes sense.

**Comment by vreuter** [comment]
This is _the_ filtration step in the WGBS pipeline. @nsheff is it fine to leave this in, or should we switch to a different filtration strategy?

**Comment by jklughammer** [comment]
Sorry for the confusion: the rna mode is not needed for normal methylation analysis. The history of this script is that it started out as a script solely for the purpose of filtering reads in a DNA methylation setting (specifically for the epignome protocol). When we worked on CORE-seq we adapted this script to basically assign the reads to whether they come from DNA (converted) or RNA (not converted, spliced etc).
